lua/core/bindings/bindings.lua

```diff
@@ -5,19 +5,26 @@ local set = vim.g
 set.mapleader = " "
 
 -- Buffer manager
-bind("n", "<Tab>", ":bn<CR>") -- Buffer Switch
-bind("n", "<C-b>", ":bd<CR>") -- Buffer Close
+bind("n", "<Tab>", ":bn<CR>")  -- Buffer Switch
+bind("n", "<C-b>", ":confirm bd<CR>")  -- Buffer Close with confirmation
 
 -- Auto comment
-bind("v", "<C-/>", ":s/^/#<CR>")
+bind("v", "<C-/>", ":s/^/#/ | s/^#//<CR>")  -- Toggle comment on selection
```

Comment:
Fix: instead of just adding comment mark at the beginning of the line, now toggles commenting